packages/integration-tests/test/wait-for-contracts-to-deploy.ts

```diff
@@ -22,5 +22,5 @@ describe('Wait for contracts to deploy', () => {
       await sleep(5000)
       return
     }
-  })
```

Comment:
Lol this is a serious timeout. Doesn't seem like a problem because I think folks will just give up before it was reached but it did stick out to me